### ISSUE TITLE
Token specific admin and user polices.

### ISSUE THREAD

**Issue by cornelinux** [privacyidea/privacyidea#4080]
What a cool software!

Actually privacyIDEA uses the policy actions "<tokentype>_otp_pin_contents" and others in the scopes `admin` and `user`.
See https://github.com/privacyidea/privacyidea/blob/95b25a1532758756d9deb28cb07a3a764af26b2d/privacyidea/lib/policy.py#L3224

Unfortunately this is not displayed in the WebUI. 

* [ ] Check which policies are missing
* [ ] Add policies to the list of actions in scope "admin" and "user", so that they are displayed in the webui. I think I recommend adding these policies in the token classes themselves. Like here https://github.com/privacyidea/privacyidea/blob/9001f3215c4940c863914fefbe96df2017b1fd3c/privacyidea/lib/tokens/pushtoken.py#L417

**Comment by nilsbehlen** [comment]
If i understand this correctly, there will be a policy option for each token type, resulting in 20+ new policies for each scope. This will clutter the policy selection pretty fast.
We might want to add a policy condition for token type (if that is possible), or change how these kind of policies are displayed.

**Comment by cornelinux** [comment]
You are right about the number of policies and your usage of the word "clutter" is correct ;-)
However, currently these policies are evaluated in the code but not displayed in the UI.

I totally agree that there should be a more sensible solution in the long run.
The question is, if we want to add this here in the short run.

**Comment by plettich** [comment]
Actually, these policies are shown in the UI:
![image](https://github.com/user-attachments/assets/4cec44b5-59e6-4aaa-bcb0-e325d4d86752)
but only for token types where `pin_scopes` are defined:
https://github.com/privacyidea/privacyidea/blob/fe1a30d3a93243a2c2e8d517cf5931904c87a377/privacyidea/lib/token.py#L2659-L2662
This is currently only set for the `SPASS` token:
https://github.com/privacyidea/privacyidea/blob/fe1a30d3a93243a2c2e8d517cf5931904c87a377/privacyidea/lib/tokens/spasstoken.py#L93
The policy is however evaluated regardless of the `pin_scopes` entry, so it works for all token types.

**Comment by jona-samuel** [comment]
It works as intended. The documentation states, "There **can** be token type-specific policies like spass_otp_pin_maxlength, spass_otp_pin_minlength, and spass_otp_pin_contents." I change this from "possible bug" to "enhancement".
